src/Models/BlogModel.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require_once 'src/Entities/Blog.php';
+require_once 'src/connectToDb.php';
+
+class BlogModel {
+    private PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function getAllPosts()
+    {
+        $query = $this->db->prepare('SELECT * FROM `blogposts` ORDER BY `posttime` DESC;');
```

Comment:
No select * in production code please. Be more specific with the fields you select.